ui/apps/dashboard/src/app/(organization-active)/(dashboard)/settings/integrations/page.tsx

```diff
@@ -1,10 +1,62 @@
 import { redirect } from 'next/navigation';
+import { auth } from '@clerk/nextjs';
 
 import { getBooleanFlag } from '@/components/FeatureFlags/ServerFeatureFlag';
+import graphqlAPI from '@/queries/graphqlAPI';
+import { getProductionEnvironment } from '@/queries/server-only/getEnvironment';
 import IntegrationsList from './integrations';
+import type { VercelIntegration } from './vercel/VercelIntegration';
+import mergeVercelProjectData from './vercel/mergeVercelProjectData';
+import { GetSavedVercelProjectsDocument } from './vercel/queries';
+
+export const notEnabledVercelIntegration: VercelIntegration = {
```

Comment:
To test this out I moved all of the fetches that were done in client-only hooks after render to an async function in the route component. useQuery, useAuth, useEnvironments, etc only work client side because we wrote them in a way that requires them to be in a client context.